plugins/modules/lke_cluster.py

```diff
@@ -177,6 +177,15 @@
     'node_pools'
 }
 
+CREATE_FIELDS: Set[str] = {
```

Comment:
Is this `CREATE_FIELDS` required to get it works? Do other modules need a similar thing or not?

Reply:
Good question! Basically, this is just a way of filtering out the meta-fields (e.g. `skip_polling`) from the actual resource fields (e.g. `node_pools`)